### PR TITLE
Add one new test for accelerometer

### PR DESCRIPTION
**Pull request by jianliu617**
Add sensor resolution and noise density new test for accelerometer.

### CODE REVIEW

**Comment by jianliu617** [comment]
@Yongjie2018, Could you please have a review for this change? Thanks.

File: sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs [hunk 1]
```diff
@@ -769,11 +779,16 @@ public async void TestEnd()
...
+            else if (testType == "ResolutionNoiseDensity")
+            {
+                CalculateResolutionNoiseDensity();
+            }
             else if (testType == "StaticAccuracy")
             {
                 CalculateStaticAccuracy();
             }
 
+
```
Comment (jianliu617) [inline sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs]: Remove this line.

File: sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs [hunk 2]
```diff
@@ -1163,6 +1178,138 @@ private void CalculatePacketLossTest()
             instruction.Text = str + "For more information, please visit:";
         }
 
+        private int GetIndexOfMaxValue(List<int> inputList)
+        {
+            int indexMax = 0;
```
Comment (Yongjie2018) [inline sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs]: Suggest to use -1 for indexMax, return value of -1 means the list is empty.

File: sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs [hunk 3]
```diff
@@ -1163,6 +1178,138 @@ private void CalculatePacketLossTest()
             instruction.Text = str + "For more information, please visit:";
         }
 
+        private int GetIndexOfMaxValue(List<int> inputList)
+        {
+            int indexMax = 0;
+            int valueMax = 0;
```
Comment (Yongjie2018) [inline sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs]: If the list is not empty, use the first element value for valueMax, and 0 for indexMax.

File: sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs [hunk 4]
```diff
@@ -1163,6 +1178,138 @@ private void CalculatePacketLossTest()
...
+                int frequency = dataList.Count / testLength[testType];
+                if (frequency > 0)
+                {
+                    finalNoiseDensity = finalNoiseDensity * 1e6 / Math.Sqrt(1.6 * frequency);
+                }
+                else
+                {
+                    finalNoiseDensity = -1;
+                }
+
```
Comment (Yongjie2018) [inline sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs:1313]: A corner case of the sensor does not report any data, how to warn the user?

File: sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs (hunk 2 above)
Comment (jianliu617) [reply sensors/Tools/SensorExplorer/Scenario0_Tests.xaml.cs]: Modified to -1. Thanks.

**Comment by jianliu617** [comment]
Tested pass on real device after review modification.

**Pull request merged**